packages/website/src/components/Provider.tsx

```diff
@@ -6,7 +6,7 @@ import * as playgroundSystem from "reakit-playground/system";
 const system = unstable_mergeSystem(bootstrapSystem, playgroundSystem, {
   palette: {
     ...bootstrapSystem.palette,
-    primary: "#6a50ee",
+    primary: "#006DFF",
     link: "#007bff",
```

Comment:
```suggestion
    link: "#006dff",
```

Reply:
This provider is used by the website? I was thinking on define a better color pallet according to accessibility. Maybe a simple color table? Like the one that Material Design uses?

Reply:
Yep! I think it's simple enough as this just extends the default palette and adds 3 more colors. But I do agree that we should review their contrast ratios and find a similar color that conforms with WCAG 2.1 AA.

Reply:
This can be a great addition to #351. I can give a look at this on the weekend. Im thinking on creating a table with the colors applied(even from Bootstrap) and their ratio :smiley: